cdp_scrapers/instances/seattle.py

```diff
@@ -178,3 +203,220 @@ def get_content_uris(self, legistar_ev: Dict) -> List[ContentURIs]:
         if len(list_uri) == 0:
             log.debug(f"No video URI found on {video_page_url}")
         return list_uri
+
+    @staticmethod
+    def get_district_image_url() -> Optional[str]:
+        """
+        Return URL for a district map image used on seattle.gov
+        Shows all district boundaries
+
+        Returns
+        -------
+        Image URL: Optional[str]
+            District map image URL
+        """
+        site_url = "https://www.seattle.gov/"
+        url = (
+            f"{site_url}cityclerk/agendas-and-legislative-resources/"
+            "find-your-council-district"
+        )
+        try:
+            with urlopen(url) as resp:
+                soup = BeautifulSoup(resp.read(), "html.parser")
+        except (URLError, HTTPError):
+            log.debug(f"Failed to open {url}")
+            return None
+
+        try:
+            # <img alt="District Map" src="Images/Clerk/DistrictsMap.jpg" ...
+            return site_url + soup.find("img", alt="District Map")["src"]
+        except (TypeError, KeyError):
+            pass
+
+        return None
```

Comment:
Commented above, but don't think we should have the same image for all districts.